packages/vscode-extension/src/commands/CreateProject.ts

```diff
@@ -42,6 +42,12 @@ export default class CreateProject extends BaseAction {
         }
     }
 
+    protected validate(): { valid: false; message: string } | { valid: true } {
```

Comment:
Is it mandatory to keep this function since it always returns true?

Reply:
Yes. I just overridden itfrom BaseAction so that there will not be a need to check if there's an active file opened to trigger the command.